### PR TITLE
readme: clarify macOS installation and update Windows folders

### PR DESCRIPTION
**Pull request by walkawayy**
#### Checklist

- [X] I have read the [coding conventions](https://github.com/LostArtefacts/TR1X/blob/master/CONTRIBUTING.md#coding-conventions)
- [X] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description
Clarify macOS installation location. Add missing Windows installation locations, fix formatting, and update shader filenames.

### CODE REVIEW

**Review by rr- — APPROVED** [review]

**Review by lahm86 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -235,7 +258,165 @@ as Notepad.
...
+        │   ├── cred1.pcx
+        │   ├── cred2.pcx
+        │   ├── cred3.pcx
+        │   ├── cut1.phd
+        │   ├── cut2.phd
+        │   ├── cut3.phd
+        │   ├── cut4.phd
+        │   ├── egypt.phd
+        │   ├── eidospc.pcx
+        │   ├── eidospc.png
```
Comment (aredfan) [inline README.md]: `eidospc.png` is removed from the data folder since the image subfolder was added.

File: README.md [hunk 2]
```diff
@@ -235,7 +258,165 @@ as Notepad.
...
+        │   ├── level4.phd
+        │   ├── level5.phd
+        │   ├── level6.phd
+        │   ├── level7a.phd
+        │   ├── level7b.phd
+        │   ├── level8a.phd
+        │   ├── level8b.phd
+        │   ├── level8c.phd
+        │   ├── titleh.pcx
+        │   ├── titleh.png
```
Comment (aredfan) [inline README.md]: `titleh.png` is removed from the data folder.

File: README.md [hunk 3]
```diff
@@ -235,7 +258,165 @@ as Notepad.
...
+        │   ├── titleh_ub.png
```
Comment (aredfan) [inline README.md]: `titleh_ub.png` is removed from the data folder.

File: README.md [hunk 4]
```diff
@@ -235,7 +258,165 @@ as Notepad.
...
+        │   ├── level3b.phd
+        │   ├── level4.phd
+        │   ├── level5.phd
+        │   ├── level6.phd
+        │   ├── level7a.phd
+        │   ├── level7b.phd
+        │   ├── level8a.phd
+        │   ├── level8b.phd
+        │   ├── level8c.phd
+        │   ├── titleh.pcx
```
Comment (aredfan) [inline README.md:311]: I'd add `titleh_ub.pcx` under `titleh.pcx` because this file is part of the collection.

**Review by aredfan — COMMENTED** [review]
I hope it's okay to submit a review for your pull request.  There's a few outdated files listed here.

**Comment by walkawayy** [comment]
Thanks @aredfan

**Pull request merged**